x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/otel_kubernetes/otel_kubernetes_panel.tsx

```diff
@@ -85,9 +85,10 @@ export const OtelKubernetesPanel: React.FC = () => {
 
   const elasticEndpointVarName = isServerless ? 'elastic_otlp_endpoint' : 'elastic_endpoint';
   const valuesFileSubfolder = isServerless ? '/managed_otlp' : '';
+  const valuesFileName = metricsOnboardingEnabled ? 'values.yaml' : 'logs-values.yaml';
```

Comment:
Should this conditional include the `isServerless`? The `logs-values.yaml` file is only available in the `managed_otlp` directory: https://github.com/elastic/elastic-agent/tree/v9.0.4/deploy/helm/edot-collector/kube-stack